wordpress-mobile/AztecEditor-iOS
# Fix placeholder images when in dark mode

Fixes #1236 

![PlaceholderDarkLight](https://user-images.githubusercontent.com/651601/70437984-b8673c00-1a84-11ea-9874-29dd6b2724c8.png)


To test:
 - Start the demo app in dark mode
 - Open the new failed media demo page
 - Check if media placeholders show correctly
 - Switch the appearance from dark to light
 - Check that placeholders update correctly
